### PR TITLE
update packageName:rangy definitions

### PR DESCRIPTION
**Pull request by ashubham**
Update type definitions for the Rangy Module.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/timdown/rangy/wiki/Text-Range-Module

### CODE REVIEW

**Comment by ashubham** [comment]
@MidnightDesign

**Comment by MidnightDesign** [comment]
Tests are missing.

**Comment by ashubham** [comment]
@MidnightDesign I cannot see any tests defined for any package types. Am I missing something ? Did you mean the travis build ? That should be passing now.

File: types/rangy/index.d.ts [hunk 1]
```diff
@@ -39,13 +39,16 @@ interface RangySelection extends Selection {
...
     getAllRanges():RangyRange[];
+    getRangeAt(idx:number):RangyRange;
     getNativeTextRange():any;
     setSingleRange(range:RangyRange):any;
     setRanges(ranges:RangyRange[]):any;
     getBookmark(containerNode:Node):any;
     moveToBookmark(bookmark:Object):any;
     saveRanges():Object;
     restoreRanges(saved:Object):any;
+    saveCharacterRanges(containerNode:Node): Object;
```
Comment (MidnightDesign) [inline types/rangy/index.d.ts]: Please keep the style consistent. (Space before `Object`)

File: types/rangy/index.d.ts [hunk 2]
```diff
@@ -62,5 +65,7 @@ interface RangyStatic {
     initialized:boolean;
     supported:boolean;
 }
-
+declare module 'rangy' {
```
Comment (MidnightDesign) [inline types/rangy/index.d.ts]: This is not tested, is it?

File: types/rangy/index.d.ts [hunk 3]
```diff
@@ -39,13 +39,16 @@ interface RangySelection extends Selection {
     refresh(checkForChanges?:boolean):any;
     toHtml():string;
     getAllRanges():RangyRange[];
+    getRangeAt(idx:number):RangyRange;
```
Comment (MidnightDesign) [inline types/rangy/index.d.ts]: This function has no test.

**Review by MidnightDesign — CHANGES_REQUESTED** [review]

File: types/rangy/index.d.ts (hunk 2 above)
Comment (ashubham) [inline types/rangy/index.d.ts]: @MidnightDesign Do you have any ideas of how this could be tested ?

File: types/rangy/index.d.ts (hunk 3 above)
Comment (ashubham) [inline types/rangy/index.d.ts]: Done

File: types/rangy/index.d.ts (hunk 1 above)
Comment (ashubham) [inline types/rangy/index.d.ts]: Done

File: types/rangy/index.d.ts (hunk 2 above)
Comment (MidnightDesign) [inline types/rangy/index.d.ts]: I don't know either. Nevermind.

**Comment by MidnightDesign** [comment]
LGTM. Thanks for the contribution!

**Review by MidnightDesign — APPROVED** [review]

**Comment by ashubham** [comment]
@DanielRosenwasser Please merge whenever you get a chance.

**Comment by DanielRosenwasser** [comment]
Sorry about that! Thanks for the PR.

**Pull request merged**